studio/schemas/documents/salaryAndBenefits.ts

```diff
@@ -1,15 +1,17 @@
 import { defineField, defineType } from "sanity";
```

Comment:
since it's deprecated and we haven't launched yet, you can delete.

Reply:
⚠️ make sure any content is deleted in PROD first ⚠️

Reply:
This is not just an issue after launch, it also makes it harder to move between versions of the codebase during development. Especially since we don't have any migration scripts. We can still remove the old schemas later when we know all the dependencies on the old schemas have been removed.

Reply:
As long as you make a note to fix